### DISCUSSION TITLE
Default dimension of transport in Application is Dimensionless

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1276] [Questions & Problems]
When creating a transport within an application, it seems that the formula is automatically of dimension "Dimensionless". There does not appear to be an option to change the dimension of the formula, but also, I would have expected the dimension to be "Amount per time".

I had created a formula and then used the workaround provided in #1275 to change the formula dimension, but once the formula dimension was successfully changed to "Amount per time", it was no longer an option for the transport.

**Comment by msevestre** [comment]
Moving this to the MoBi repository
